react/week3/fetchtodos/src/FetchTodos.js

```diff
@@ -0,0 +1,176 @@
+import React, { useEffect, useState } from "react";
+import PropTypes from "prop-types";
+// this componet is for styling the border
+const TodoComponent = (props) => {
+  return (
+    <div
+      style={{
+        backgroundColor: "yellow",
+        border: "2px solid black",
+        width: "700px",
+        marginLeft: "250px",
+        marginBottom: "20px",
+      }}
+    >
+      {props.children}
+    </div>
+  );
+};
+
+const API_URL =
+  "https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw";
+
+function TodoList({ id, description, deadline, removeTodo }) {
```

Comment:
This component seems to be a `TodoItem` and the one you've named `RenderTodos` seems to be the actual `TodoList`.

If you want to make it easier to read through, I'd split the `BorderWrapper`, `TodoItem`, and `TodoList` components into each their file :)